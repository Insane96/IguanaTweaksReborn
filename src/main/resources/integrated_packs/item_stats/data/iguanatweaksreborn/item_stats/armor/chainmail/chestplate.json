{
	"item": "minecraft:chainmail_chestplate",
	"durability": 40,
	"armor": 4.2,
	"modifiers": [

	]
}