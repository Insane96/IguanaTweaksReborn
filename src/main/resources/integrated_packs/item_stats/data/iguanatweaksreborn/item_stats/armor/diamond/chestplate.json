{
	"item": "minecraft:diamond_chestplate",
	"durability": 192,
	"armor": 5,
	"modifiers": [

	]
}