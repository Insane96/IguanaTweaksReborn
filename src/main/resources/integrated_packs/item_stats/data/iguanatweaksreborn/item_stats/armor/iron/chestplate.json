{
	"item": "minecraft:iron_chestplate",
	"durability": 112,
	"armor": 3,
	"regenerating_absorption": 1.5,
	"modifiers": [

	]
}