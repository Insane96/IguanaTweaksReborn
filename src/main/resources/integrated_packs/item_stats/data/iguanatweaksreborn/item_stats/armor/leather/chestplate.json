{
	"item": "minecraft:leather_chestplate",
	"durability": 64,
	"armor": 1.5,
	"regenerating_absorption": 2,
	"modifiers": [

	]
}