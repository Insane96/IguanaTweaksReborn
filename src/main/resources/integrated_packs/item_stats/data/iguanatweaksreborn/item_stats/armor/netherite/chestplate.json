{
	"item": "minecraft:netherite_chestplate",
	"durability": 176,
	"armor": 5.5,
	"modifiers": [

	]
}