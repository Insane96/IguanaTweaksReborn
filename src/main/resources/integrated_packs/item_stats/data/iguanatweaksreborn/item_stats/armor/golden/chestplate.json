{
	"item": "minecraft:golden_chestplate",
	"durability": 64,
	"armor": 1.5,
	"modifiers": [

	]
}